license.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
GDEDBDNH

Reply:
> GDEDBDNH

Bro reply

Reply:
i don't know what is the license bro

Reply:
can u give it to me if you know?

Reply:
pls provide license key i'll give u my eth if the application works